schemars/examples/schemars_attrs.schema.json

```diff
@@ -26,6 +27,13 @@
       "format": "int32",
       "maximum": 10.0,
       "minimum": 1.0
+    },
+    "myVecStr": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "pattern": "^x$"
```

Comment:
This isn't the actual output when the schema is generated - you can verify this by running `update-examples.sh` in the project root